apps/avatax/vercel.json

```diff
@@ -1,21 +1,3 @@
 {
-  "$schema": "https://openapi.vercel.sh/vercel.json",
-  "functions": {
-    "src/pages/api/webhooks/checkout-calculate-taxes.ts": {
-      "memory": 320,
-      "maxDuration": 25
-    },
-    "src/pages/api/webhooks/order-calculate-taxes.ts": {
-      "memory": 320,
-      "maxDuration": 25
-    },
-    "src/pages/api/webhooks/order-cancelled.ts": {
-      "memory": 320,
-      "maxDuration": 25
-    },
-    "src/pages/api/webhooks/order-confirmed.ts": {
-      "memory": 320,
-      "maxDuration": 25
-    }
```

Comment:
issue: Is this safe that we are using default memory and duration? IMHO we should have a safe limit for duration, 25s is enough for sync webhooks because over 20s that Saleor will timeout request anyway

Memory is used for cost savings since Vercel is charging for that as well.

By default Vercel uses 1024 MB of memory and 15s max duration: https://vercel.com/docs/functions/limitations ❗

If increasing memory improves cold starts then that's fine, but I think we should leave `maxDuration`

Reply:
Update: This configuration is in Vercel dashboard, nowadays Vercel has 3 settings for lamdbas, if we specify this low memory we also impact our CPU configuration.